### ISSUE TITLE
[FEATURE] - Support for @example from JSDoc

### ISSUE THREAD

**Issue by lexon0011** [compodoc/compodoc#90]
We use the @example tag in our comments, like

```
/**
 * Solves equations of the form a * x = b
 * @example
 * // returns 2
 * globalNS.method1(5, 10);
 * @example
 * // returns 3
 * globalNS.method(5, 15);
 * @returns {Number} Returns the value of x for the equation.
 */
```
see [JSDoc](http://usejsdoc.org/tags-example.html)

Is it possible to include this example-tags to the generated documentation?

Thanks!

**Comment by lexon0011** [comment]
I try it with the latest version, but the @example block is not shown in generated documentation.

**Comment by vogloblinsky** [comment]
Works fine in the demo :

https://github.com/compodoc/compodoc-demo-todomvc-angular2/blob/master/src/app/shared/services/todo.store.ts#L46

https://compodoc.github.io/compodoc-demo-todomvc-angular2/injectables/TodoStore.html

**Comment by lexon0011** [comment]
Yes you are right, but we have the problem, that we use the @example attribute on a classes like:

```
/**
 * Set all todos status (completed or not)
 *
 * @example
 * // set all at completed
 * TodoStore.setAllTo(true);
 *
 * @example
 * // set all at not completed
 * TodoStore.setAllTo(false);
 *
 */
@Component({
    selector: "error-alert",
    template: `
        <b>test</b>
    `
})
export class ErrorAlertComponent implements OnInit, OnDestroy, AfterViewInit {
    ...
}
```

In addition, I have one other question: We want to change the style for the sample code. But the div, which includes the code, has no class:
```
<div><p>// set all at completed<br>TodoStore.setAllTo(true);</p></div>
```

Is it possible to add a class attribute (e.g. `class="jsdoc-example"`) to the div. For example:
```
<div class="jsdoc-example"><p>// set all at completed<br>TodoStore.setAllTo(true);</p></div>
```

Thank you very much!

**Comment by vogloblinsky** [comment]
JSDoc @example tag is only supported for class methods.
I will enhanced the main description of components for supporting it too.

**Comment by lexon0011** [comment]
Thank you, you're great!

**Comment by lexon0011** [comment]
Do you know, when you will implement this enhancement? Thank you very much!

**Comment by vogloblinsky** [comment]
It

**Comment by vogloblinsky** [comment]
Fixed in 0.0.41
